bootstrap/src/main/java/org/jboss/windup/bootstrap/listener/GreetingListener.java

```diff
@@ -23,20 +26,37 @@
 {
     private final Logger logger = Logger.getLogger(getClass().getName());
 
+    private static String WINDUP_BRAND_NAME_LONG = "";
+    private static String WINDUP_BRAND_DOCUMENTATION_URL = "";
+
     public GreetingListener()
     {
     }
 
+    // We need to have this file here as a Workaround for not being able
+    // to consume org.jboss.windup.util.Util because of NotClassFoundException in windup-distribution
```

Comment:
I've verified and this is the result:
- This PR, which is synchronized with master branch, generates the ClassNotFoundException even if it is build with JDK 8
- It seems the problem has been introduced by https://github.com/windup/windup/commit/29d79e9c2deacd87dfec00f6adf047d5d46e36e8 because I tested that if I use a previous version previous to this commit and then apply my changes, the ClassNotFoundException doesn't appear

I was not able to determine what is the reason behind it though.

Reply:
@jmle  As discussed internally, the ClassNotFoundException error might be addressed by another ticket https://issues.redhat.com/browse/WINDUP-3283 in order to move this change forward. May I ask you to review this PR again, please? It has been a while since this PR has been submitted and it might be worth checking it again.